packages/linux-firmware/firmware-imx/package.mk

```diff
@@ -22,7 +22,7 @@ makeinstall_target() {
   mkdir -p $INSTALL/$(get_full_firmware_dir)/imx/sdma
     cp -P firmware/sdma/sdma-imx6q.bin $INSTALL/$(get_full_firmware_dir)/imx/sdma
 
-  mkdir -p $INSTALL/$(get_full_firmware_dir)/vpu
```

Comment:
you don't really need to change this, the kernel looks for the vpu dir.

https://github.com/torvalds/linux/blob/f01c30de86f1047e9bae1b1b1417b0ce8dcd15b1/drivers/media/platform/coda/coda-common.c#L3057-L3062

Reply:
Yes you are right, I just wanted to get rid of:
```
coda 2040000.vpu: Direct firmware load for vpu_fw_imx6q.bin failed with error -2
coda 2040000.vpu: Using fallback firmware vpu/vpu_fw_imx6q.bin
```

Reply:
ok, I can live with that change.